firmware/targets/f7/furi_hal/furi_hal_subghz.h

```diff
@@ -5,6 +5,8 @@
 
 #pragma once
 
+#include <lib/subghz/devices/preset.h>
```

Comment:
Move to separate entity

Reply:
General idea is to simplify furi_hal_subghz that it doesn't know anything about specific presets.
Instead provide methods to load raw preset data.

On device level implement loading preset data by enum

Reply:
+fix